x-pack/test/functional/apps/machine_learning/anomaly_detection/categorization_job.ts

```diff
@@ -0,0 +1,392 @@
+    });
+
+    it('job cloning sets the timerange', async () => {
+      await ml.jobWizardCommon.clickUseFullDataButton(
+        'Apr 5, 2019 @ 11:25:35.770',
+        'Nov 21, 2019 @ 06:01:13.914'
+      );
+    });
+
+    it('job cloning displays the event rate chart', async () => {
+      await ml.jobWizardCommon.assertEventRateChartExists();
+      await ml.jobWizardCommon.assertEventRateChartHasData();
+    });
+
+    it('job cloning displays the pick fields step', async () => {
+      await ml.jobWizardCommon.advanceToPickFieldsSection();
+    });
+
+    it('job cloning pre-fills field and aggregation', async () => {
+      await ml.jobWizardCommon.assertCategorizationDetectorTypeSelectionExists();
+      await ml.jobWizardCommon.assertCategorizationDetectorTypeSelected(detectorTypeIdentifier);
+    });
+
+    it('job cloning pre-fills the bucket span', async () => {
+      await ml.jobWizardCommon.assertBucketSpanInputExists();
+      await ml.jobWizardCommon.assertBucketSpanValue(bucketSpan);
+    });
+
+    it('job cloning displays the job details step', async () => {
+      await ml.jobWizardCommon.advanceToJobDetailsSection();
+    });
+
+    it('job cloning does not pre-fill the job id', async () => {
+      await ml.jobWizardCommon.assertJobIdInputExists();
+      await ml.jobWizardCommon.assertJobIdValue('');
+    });
+
+    it('job cloning inputs the clone job id', async () => {
+      await ml.jobWizardCommon.setJobId(jobIdClone);
+    });
+
+    it('job cloning pre-fills the job description', async () => {
+      await ml.jobWizardCommon.assertJobDescriptionInputExists();
+      await ml.jobWizardCommon.assertJobDescriptionValue(jobDescription);
+    });
+
+    it('job cloning pre-fills job groups', async () => {
+      await ml.jobWizardCommon.assertJobGroupInputExists();
+      await ml.jobWizardCommon.assertJobGroupSelection(jobGroups);
+    });
+
+    it('job cloning inputs the clone job group', async () => {
+      await ml.jobWizardCommon.assertJobGroupInputExists();
+      await ml.jobWizardCommon.addJobGroup('clone');
+      await ml.jobWizardCommon.assertJobGroupSelection(jobGroupsClone);
+    });
+
+    it('job cloning opens the additional settings section', async () => {
+      await ml.jobWizardCommon.ensureAdditionalSettingsSectionOpen();
+    });
+
+    it('job cloning persists custom urls', async () => {
+      await ml.customUrls.assertCustomUrlItem(0, 'check-kibana-dashboard');
+    });
+
+    it('job cloning persists assigned calendars', async () => {
+      await ml.jobWizardCommon.assertCalendarsSelection(['wizard-test-calendar']);
+    });
+
+    it('job cloning opens the advanced section', async () => {
+      await ml.jobWizardCommon.ensureAdvancedSectionOpen();
+    });
+
+    it('job cloning pre-fills the model plot switch', async () => {
+      await ml.jobWizardCommon.assertModelPlotSwitchExists();
+      await ml.jobWizardCommon.assertModelPlotSwitchCheckedState(true);
```

Comment:
See comment above: the default value for this switch is `false` (and we currently don't change it during the test).
Would also be good to assert that the switch is disabled in case of the `rare` detector.